lib/chef/powershell.rb

```diff
@@ -66,10 +66,10 @@ def error!
 
     protected
 
-    def exec(script)
+    def exec(script, timeout)
       FFI.ffi_lib @dll
-      FFI.attach_function :execute_powershell, :ExecuteScript, [:string], :pointer
-      execution = FFI.execute_powershell(script).read_utf16string
+      FFI.attach_function :execute_powershell, :ExecuteScript, [:string, :int], :pointer
```

Comment:
@lamont-granquist not sure about this but are these ffi methods memoized? i.e. initializing `Powershell` (calling `exec`) multiple times won't be a problem?